salesagility/SuiteCRM
# Scheduled Reports: Fix report name relation and popup search

## Description

Both problems resulted in the popup search not showing anything and erroring out,
the report name being empty in the list view and sorting by the report name would
break the module because it would throw SQL errors without any way to recover and undo
the sorting.

This includes the following fixes for all of the above:

* Fix the link id to point to the link
* Rename the "aor_report_name" field in the search popup to match the vardef
* Remove the "email" field in the search popup which doesn't exist in the vardef.

## Motivation and Context

I want to use the Scheduled Reports module :)

## How To Test This

1) Create a Report
2) In the scheduled reports subpanel try to search for an existing one **(was broken)**
3) Create a scheduled report for the report
4) Go to the scheduled report list view and see the associated report name in the table **(was broken)**
5) Try to sort the scheduled reports by related report name in the listview **(was broken)**
6) Remove the scheduled report from the report
7) Re-add it again through the report subpanel search popup **(was broken)**

## Types of changes

### Final checklist
